draft-ietf-sedate-datetime-extended.md

```diff
@@ -789,7 +789,8 @@ extensions still to be defined.
 Where consistent interpretation between multiple actors is required
 for security purposes (e.g., where timestamps are embedded as
 parameters in access control information), only such extensions can be
-employed that have a defined resolution of such inconsistent data.
+employed that have a well-understood and shared resolution of such
```

Comment:
Grammar: s/that/as/.

That's "only such [whatever] as have [what's needed]"; contrast with "you can use [a thing] that has [what's needed], but not one that lacks it."

Reply:
Don't feel too strongly about this either way, but that feels better here.

Perhaps `s/that/which` ("...only such extensions can be employed which have a...")

Reply:
(Well, at this point, I'd s/that/which/ but it'll do.)